src/lib/Toast/Toast.svelte

```diff
@@ -0,0 +1,149 @@
+<script lang="ts">
+    import { fly } from 'svelte/transition';
+  
+    import { defaultToastProperties, type ToastProperties } from './properties';
+    import errorIcon from './error.svg';
+    import closeIcon from './close.svg';
+    import infoIcon from './info.svg'
+    import { onMount } from 'svelte';
+  
+    export let properties: ToastProperties = defaultToastProperties;
+  
+    let visible = false;
+  
+    $: leftIcon = properties.leftIcon ? properties.leftIcon : '';
+  
+    properties = {
+      ...defaultToastProperties,
+      ...properties
+    };
+  
+    onMount(() => {
+      visible = true
+      setTimeout(() => {
```

Comment:
collect the timeOut response and clear it on onDestroy

Reply:
updated.